build.gradle.kts

```diff
@@ -80,6 +80,9 @@ configurations {
 dependencies {
     shadowed("com.squareup.retrofit2:retrofit:2.7.1")
     shadowed("com.squareup.retrofit2:converter-gson:2.7.1")
+    shadowed("io.github.alexo:retrier:0.1") {
+        exclude("org.mockito")  // https://github.com/alexo/retrier/pull/1
+    }
```

Comment:
I've had a good experience with https://jodah.net/failsafe/ for this in the past. It seems more mature, too. Should we switch?

Reply:
I didn't know it. I will take a look.

Reply:
It's very nice. That's pity that is was in my search results at the end of the second page :-/. Replaced.